plugins/org.jkiss.dbeaver.ext.sqlite/plugin.xml

```diff
@@ -17,6 +17,101 @@
                 label="SQLite"
                 icon="icons/sqlite_icon.png"
                 dialect="sqlite">
+
```

Comment:
Why copy-pasting entire tree with all localizations? 
It doesn't look related to STRICT keyword

Reply:
Yes, but I hadn't figured out how to make an injection...